pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -279,7 +281,10 @@
     }
 
     public static final String DEFAULT_METRICS_PREFIX = "pinot.server.";
-    public static final boolean DEFAULT_METRICS_GLOBAL_ENABLED = false;
+    public static final String CONFIG_OF_ENABLE_TABLE_LEVEL_METRICS = "pinot.server.enableTableLevelMetrics";
```

Comment:
I think you are trying to keep the new configs you introduced in the same place. It is unfortunate that we don't have a BrokerConf (as yet), but let us keep the server-specific configs in the ServerConf file instead of moving that up to common constants.

Reply:
Other than keeping similar config in one place, there's also this todo in server conf: `TODO: Replace with constants in CommonConstants`.
Having said that the main reason to move to CommonConstant is that in the existing code, the default value for enabling table level metrics is defined in two places: ServerConf and CommonConstants. ServerConf lives in pinot-broker module and CommonConstant in pinot-common. Both are needed as ServerConf is used in ServerInstance defined in pinot-broker module and CommonConstant is used in ServerMetrics which is defined in pinot-common module.
Having it only defined in CommonConstant removes this duplicity.